crowbar_framework/app/models/node.rb

```diff
@@ -903,6 +903,8 @@ def wait_for_script_to_finish(script, seconds, args = [])
     cmd = script
     cmd += " " + args.join(" ") unless args.empty?
 
+    raise "Script #{script} is not present at the node #{@node.name}." unless file_exist?(script)
```

Comment:
I think it's `s/at/on/` but you might want to check back with a native speaker.

Reply:
I would probably say "on" myself too, but I stole this message from @jsuchome :)

Reply:
I've asked Gary Smith who's a native speaker :-) He recommended using 'on' instead of 'at' in this case.

Reply:
Oh, thanks for assessing my English ;-)

(Feel free to correct it of course)